packages/approval-controller/src/ApprovalController.test.ts

```diff
@@ -434,6 +473,30 @@ describe('approval controller', () => {
       approvalController.clear(new EthereumRpcError(1, 'clear'));
       expect(approvalController.getTotalApprovalCount()).toStrictEqual(0);
     });
+
+    it('gets the total approval count with type excluded from rate limiting', () => {
+      const approvalController = new ApprovalController({
+        messenger: getRestrictedMessenger(),
+        showApprovalRequest: sinon.spy(),
+        typesExcludedFromRateLimiting: ['type0'],
+      });
+      expect(approvalController.getTotalApprovalCount()).toStrictEqual(0);
+
+      const addWithCatch = (args: any) =>
+        approvalController.add(args).catch(() => undefined);
```

Comment:
We are setting this method 3 times in this file. Can't we just create a higher level `addWithCatch` and pass an instance of the approvalController in the args? Something like:
```js
const addWithCatch = ({
  approvalController,
  args,
}: {
  approvalController: ApprovalController,
  args: any,
}) => approvalController.add(args).catch(() => undefined);
```